presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -153,3 +153,45 @@ Expression              Meaning
 ====================    ===========
 
 ``ANY`` and ``SOME`` have the same meaning and can be used interchangeably.
+
+LIKE
+----
+The LIKE operator is used to match a specified character pattern in a string. Patterns can contain
+regular characters as well as wildcards. Wildcard characters can be escaped. Matching is case sensitive.
```

Comment:
Also, say '..can be escpaed using the single character specified for the ESCAPE parameter'

Reply:
Sorry - also NULL behavior?